src/plugins/unified_search/public/autocomplete/providers/value_suggestion_provider.test.ts

```diff
@@ -189,24 +189,28 @@ describe('FieldSuggestions', () => {
       };
 
       await getValueSuggestions({
+        // @ts-ignore
         indexPattern: customIndexPattern,
```

Comment:
I was wondering if we could remove this `ts-ignore`. I think if we cast `customIndexPattern` to `DataView` , eg: `indexPattern: customIndexPattern as DataView` this could be possible.